canonicalwebteam/launchpad/models.py

```diff
@@ -204,3 +204,24 @@ def build_snap(self, snap_name):
             )
 
         return True
+
+    def get_snap_build(self, snap_name, build_id):
+        """
+        Return a Snap Build from the Launchpad API
+        """
+        lp_snap = self.get_snap_by_store_name(snap_name)
+
+        return self._request(
```

Comment:
this just get's the snap doesn't it? Doesn't seem to use the build_id?

Reply:
Sorry, I forgot to update the URL...